src/tutorials/0007-verifying-storage-on-filecoin/01.js

```diff
@@ -0,0 +1,28 @@
+// Question must be a string
+const question = "Where does trust come from in Filecoin's storage network?"
```

Comment:
```suggestion
const question = "Without a centralized company, how can we trust that data is being stored correctly on the Filecoin storage network?"
```

Reply:
@mishmosh on our call we had landed on "How can we trust that data is being stored correctly on the Filecoin network?" Is that okay or are you still hoping for this edit? (The centralized thing is called out in a wrong answer.)

Reply:
You can ignore this suggestion.